src/pages/NoticeTree/index.tsx

```diff
@@ -29,36 +31,47 @@ const NoticeTree = () => {
 	};
 
 	const getNoticeMessages = useCallback(async () => {
-		const defaultNotices = await getNotices();
-		if (defaultNotices !== undefined) {
-			const newNoticeMessages = defaultNotices.messages.map((message, idx) => ({
-				...message,
-				width: Math.floor(Math.random() * 100),
-				height: Math.floor(Math.random() * 100),
-				messageIndex: idx + 1,
-			}));
-			setNoticeMessages(newNoticeMessages);
-			setTotalUnreadMessageCount(defaultNotices.totalUnreadMessageCount);
-		} else {
-			modalHandler();
+		try {
+			const defaultNotices = await getNotices();
+			if (defaultNotices.messages.length > 0) {
+				const newNoticeMessages = defaultNotices.messages.map((message, idx) => ({
+					...message,
+					width: Math.floor(Math.random() * 100),
+					height: Math.floor(Math.random() * 100),
+					messageIndex: idx + 1,
+				}));
+				setNoticeMessages(newNoticeMessages);
+				setTotalUnreadMessageCount(defaultNotices.totalUnreadMessageCount);
+			} else {
+				if (!sessionStorage.getItem('noticeTree-read')) {
+					modalHandler();
+					sessionStorage.setItem('noticeTree-read', 'true');
+				}
+			}
+		} catch {
+			setErrorToastText('네트워크에러');
 		}
 	}, []);
 
 	const updateReadMessageHandler = async (messageId: number, selectedIdx: number) => {
-		if (noticeMessages) {
-			if (messageId > 0) {
-				const isRead = await updateReadMessage(messageId);
+		try {
+			if (noticeMessages) {
+				if (messageId > 0) {
+					const isRead = await updateReadMessage(messageId);
 
-				!isRead && setUnreadCount((prev) => prev - 1);
-			}
+					!isRead && setUnreadCount((prev) => prev - 1);
+				}
 
-			showMessageHandler(true);
-			setSelectedMessage(noticeMessages[selectedIdx]);
+				showMessageHandler(true);
+				setSelectedMessage(noticeMessages[selectedIdx]);
 
-			const data = [...noticeMessages].filter((arr, idx) => {
-				return idx !== selectedIdx;
-			});
-			setNoticeMessages(data);
+				const data = [...noticeMessages].filter((arr, idx) => {
+					return idx !== selectedIdx;
+				});
```

Comment:
```suggestion
				const data = [...noticeMessages].filter((arr, idx) => idx !== selectedIdx);
```

화살표 함수를 사용하셨으니 한줄로도 가능하겠네요!